examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        elif prepro.endswith('core'):
</pre><p>This kind of argument parsing by pattern matching is not ideal. There are better alternatives e.g. <code>prepro</code>   could be Union[None, int], or just int (0 would return the original df).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
So the idea is to have `prepro` as one of the following: original, N-core, N-filter where core and filter works in different ways.

Reply:
? There are only two cases in the code:
```
    if prepro == "original":
        return df
    
    elif prepro.endswith('core'):
```

Reply:
Modified the function to have only integer input.